apps/snitch_core/test/data/model/product_review_test.exs

```diff
@@ -55,6 +56,20 @@ defmodule Snitch.Data.Model.ProductReviewTest do
     assert result.review_count == 2
   end
 
+  @tag rating_option_count: 2
+  describe "fetch_rating_option/1" do
+    test "return a valid rating_option", context do
+      %{rating_options: rating_options} = context
+      [rating_option_1 | [rating_option_2 | _]] = rating_options
```

Comment:
this can be written simply as [rating_option_1, rating_option_2] since you are inserting only two rating options and you are controlling it using tags.